deploy-codeintel-db.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+set -e
+
+# Description: PostgreSQL database for code intelligence data.
+#
+# Disk: 128GB / persistent SSD
+# Network: 100mbps
```

Comment:
pretty slow for the amnt of data this DB has - maybe you want 1Gbps ?

Reply:
I'm not even sure where this comes from. Is this the default docker network bandwidth?

Reply:
The machine it’s served on, docker and/or other container management platform restrictions (remember this deploy type is just saying “If you were to arbitrarily deploy these containers somewhere and point them out each other this is what they would need”)

For example, say someone was deploying this container and a bunch of others on a single machine with 10Gbps of network bandwidth itself - what network bandwidth can we expect to have reserved for this container?

I would put 1Gbps or 3Gbps here based on what I know about this service

Reply:
Oh it's just a suggestion? I thought it was related to the attached network somehow.

Reply:
Just a suggestion, but someone may use this info to restrict your network speed to what you write here in the future - so..

Reply:
Updated.